includes/class-wc-subscriptions-renewal-order.php

```diff
@@ -55,11 +55,23 @@ public static function trigger_renewal_payment_complete( $order_id ) {
 	 * @return mixed If the renewal order did replace a failed order, the ID of the fail order, else false
 	 */
 	public static function get_failed_order_replaced_by( $renewal_order_id ) {
-		global $wpdb;
 
-		$failed_order_id = $wpdb->get_var( $wpdb->prepare( "SELECT post_id FROM $wpdb->postmeta WHERE meta_key = '_failed_order_replaced_by' AND meta_value = %s", $renewal_order_id ) );
+		// Get orders where order meta '_failed_order_replaced_by' = $renewal_order_id
+		$failed_orders = wcs_get_orders_with_meta_query(
+			array(
```

Comment:
Small nit-pick but we're using the old `array()` syntax here.

Reply:
Thanks! I've updated to short `[]` syntax, which is consistent with the rest of this codebase.

This confuses me a little since the [WP coding standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays) say: "Arrays must be declared using long array syntax."

I'm assuming that WooCommerce standards allow it?